styles/alert.css

```diff
@@ -0,0 +1,47 @@
+.alert-container {
+    flex-direction: column;
```

Comment:
'display: flex' is not declared here. without that other flex, properties will not apply.

Reply:
Will do it. Thanks for pointing it out.